packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -83,7 +83,7 @@ export class ArtifactCreator {
       const analysisResults = await this.analyze(
           screenshot, goldens, scenario, dimensions, analysisThresholds);
 
-      const scenarioRecord = Object.assign({analysisResults}, scenario);
+      const scenarioRecord = {analysisResults, scenario};
 
```

Comment:
This may not be necessary, but i just want to keep the analysis result consistent with the ScenarioRecord interface i just created. i run the full fidelity test and found no errors. Also, correct me if im wrong, i think the scenario property is not necessary since i did find it's used anywhere